### PR TITLE
i2pd: add -devel subpackage

### PR DESCRIPTION
**Pull request by kevcrumb**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-musl
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-glibc
  - aarch64-musl
  - aarch64-glibc

### CODE REVIEW

File: srcpkgs/i2pd/template [hunk 1]
```diff
@@ -27,6 +27,13 @@ case "${XBPS_TARGET_MACHINE}" in
 esac
 
 do_install() {
+	# Static library and headers for the -devel package
+	vinstall libi2pd.a 644 usr/lib
+	for f in libi2pd/*.h ; do
+		vinstall $f 644 usr/include
+	done
```
Comment (classabbyamp) [inline srcpkgs/i2pd/template:34]:
```suggestion
	vcopy "libi2pd/*.h" usr/include
```

may require `vmkdir usr/include` too

File: srcpkgs/i2pd/template [hunk 2]
```diff
@@ -27,6 +27,13 @@ case "${XBPS_TARGET_MACHINE}" in
 esac
 
 do_install() {
```
Comment (classabbyamp) [inline srcpkgs/i2pd/template:29]: what about the shared library? you should be able to build that with an additional build target (`make_build_target="all api"` i think should work)

**Comment by paper42** [comment]
this is not really adding a package, so the commit should say something like "i2pd: add -devel subpackage"

File: srcpkgs/i2pd/template [hunk 3]
```diff
@@ -37,5 +44,15 @@ do_install() {
 	vmkdir usr/share/i2pd
 	vcopy contrib/certificates usr/share/i2pd
 	vmkdir var/lib/i2pd
-	ln -s /usr/share/i2pd/certificates ${DESTDIR}/var/lib/i2pd/certificates
+	ln -sf /usr/share/i2pd/certificates ${DESTDIR}/var/lib/i2pd/certificates
+}
+
+
```
Comment (classabbyamp) [inline srcpkgs/i2pd/template:50]:
```suggestion
```

File: srcpkgs/i2pd/template (hunk 2 above)
Comment (classabbyamp) [reply srcpkgs/i2pd/template:30]: ok, after looking at the makefile more, it seems they don't version the shared library, so there's not much point in using it, I think.

**Comment by kevcrumb** [comment]
bumping just in case this got out of sight

**Pull request merged**